src/settings.rs

```diff
@@ -299,10 +299,8 @@ impl TryFrom<u64> for KeyslotsSize {
         if v > Self::MAX_MB || v % Self::FOUR_KB != 0 {
             return Err(LibcryptErr::InvalidConversion);
         }
-        // Number of 4k blocks
-        let four_k_blocks = v / Self::FOUR_KB;
 
-        Ok(KeyslotsSize(four_k_blocks))
+        Ok(KeyslotsSize(v))
```

Comment:
It looks to me like docs on the definition of KeySlotSize have gotten out of sync. Based on the rest of the code, KeyslotSize is the size in _bytes_.

Reply:
Resolved.